### DISCUSSION TITLE
Product association problem

### DISCUSSION THREAD

**Discussion by Danieleeffe1** [OpenMage/magento-lts#2213] [Questions & Answers]
OpenMage 2.0.13

1 I create a product
2 I associate related products
3 I don't see some related products on the product page

I noticed a strange problem with the "related products" function

I associate related products and they are not seen
I tried unsuccessfully:
inventory management disabled
forced the status to "visible"
forced status to "available"
clean cache
clean cache fpc redis
disabled fpc redis
disabled fpc openmage
disabled cache
reindexed every time I retried.

in the end I managed but I was forced to duplicate the products I wanted to associate.

is it a bug?

ps:

the problem seems to depend on inventory management.

I repeat:
openmage is set to not manage stocks.
all products are set to not manage inventory.
all products are set as AVAILABLE

**Comment by addison74** [comment]
Please try to give a coherent format to a report. On my word that it was very difficult for me to read it.

Did you check this issue with an OpenMage installed from scratch? I will try to reproduce the problem.

**Comment by Danieleeffe1** [comment]
I thank you for the answer

I'm sorry for the way I presented the problem

it is not easy for me to exhibit it as you wish.
I can tell you
I try to take a picture of the situation:

Server:
Ubuntu 20.04 64 bit
PHP7.4
Redis 6.0.6
all php dependencies satisfied

Open wizard
Initial installation 20.0.13

Connected to DB MySql 8.0.28


I exported the db of an openmage installation on another server
I installed openmage 20.0.13 and connected to the DB

I have installed all the missing plug-ins
I copied all the missing images
I copied all the installation files and saved them in a "tmp" folder.
I copied the Magento 1.9.4.5 files to the OpenMage root since I need the backup function.
I copied the files from the "tmp" folder to the root
Everything works fine
In admi> configuration> catalog> inventory I set "manage inventory" to "yes".
In the products I started a Bach process to set all products as "not available"

after
I set "Manage inventory to" NO "
In the products I started a Bach process to set all products as "available"

I cleaned the cache
I performed a reindex

The problem occurs only with the last 3 simple products inserted before the transfer of the new server
.
I have created bundled products
I have associated the child products and I have blocked the choice of the number of products.
In the "Associated products" tab I have searched and associated all the child products of the bundle to give the customer a shortcut to the page of the child products that are visible in the front-end.
these child products do not allow the publication of the Bundle, it seems that the child products are "not available", but if I check their product page they are available.
If I run an "inventory" check of the child products the "inventory management" is "no".
these child products cannot be associated with any type of product,
I have cloned these child products and the clones are not causing any problems.

**Comment by addison74** [comment]
I tried to replicate this issue without any success based on latest OM-20.0.14. If anyone wants to test is welcome. In 2 weeks from now I will transfer to Discussions -> Q&A.
